bedrock-migration/templates/bedrock-migration-job.yaml

```diff
@@ -24,34 +22,65 @@ spec:
             servicesNamespace={{ .Values.global.instanceNamespace }}
             namespaces=$(oc get cm namespace-scope -n $operatorNamespace -o jsonpath="{.data.namespaces}")
 
+            
             nssSub=$(oc get subscription.operators.coreos.com -n "$operatorNamespace" -o jsonpath="{.items[?(@.spec.name=='ibm-namespace-scope-operator')].metadata.name}")
-            nssCSV=$(oc get --ignore-not-found sub $nssSub -n $operatorNamespace -o jsonpath='{.status.currentCSV}')
-            nssVersion=$(oc get --ignore-not-found csv $nssCSV -n $operatorNamespace -o jsonpath='{.spec.version}')
-
+            if [[ -z $nssSub ]]; then
+              echo "NamespaceScope Subscription not present in operator namespace $operatorNamespace, skipping deleting Subscription and CSV."
+            else
+              nssCSV=$(oc get --ignore-not-found sub $nssSub -n $operatorNamespace -o jsonpath='{.status.currentCSV}')
```

Comment:
please use fully qualified Subscription name - the `sub` and `subscription` is amgibuous